### DISCUSSION TITLE
[Feature] Shared Tags - Make tags shareble between users.

### DISCUSSION THREAD

**Discussion by malavolti** [immich-app/immich#12356] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to start by expressing my gratitude for the excellent application provided and its continuous improvements.

I would like to ask if it would be possible to create shared tags, where users can choose the name of the tag and the people with whom to share them. I realize this might not be easy since it would mean making images shareable that would otherwise remain private without the tag, but I still wanted to make my request.

The shared tags could even be limited to shared albums only.

This would allow users to create a kind of "Favorites" for those who share the tag.

I hope I have explained myself clearly enough and that this request is not too difficult to implement.

Thank you very much again!

Marco

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
I think this is a duplicate of #6350? #1673 will also make some of this possible.

**Comment by daerlnaxe** [comment]
I'm not sure about the current behavior, but I'd like to share my use case. I've created tags for my albums and shared them, but other people can't see those tags. Being able to share tags along with albums would be very useful, as it would save everyone from having to re-add the same tags manually.

Imagine a family of 5–6 people using Immich. You might set up a shared admin account to prevent accidental deletions, but right now, i can't see a  way to share metadata like tags.

**Comment by bluratelier-cyber** [comment]
That´s the feature that we are also missing. We are using the immich as the tool to keep a grasp of the photos in our tattoo studio - with studio as the admin account + tattoo artists as partners. We have a tagging script with LLM for the images, but the tags never reach the artists :(
